app/validators/waste_exemptions_engine/main_person_validator.rb

```diff
@@ -5,9 +5,21 @@ class MainPersonValidator < PersonValidator
     private
 
     def validate_number_of_people(record)
-      return if record.enough_main_people?
+      return false unless enough_people?(record)
+
+      not_too_many_people?(record)
+    end
+
+    def enough_people?(record)
+      return true if record.enough_main_people?
 
       record.errors.add(:base, :not_enough_main_people, count: record.minimum_main_people)
     end
+
+    def not_too_many_people?(record)
+      return false unless record.too_many_main_people?
+
+      record.errors.add(:base, :too_many_main_people, count: record.minimum_main_people)
```

Comment:
I struggled with this as I wanted to maintain the existing pattern which is 'is the condition valid?' The condition is 'is the number of people less than the maximum + 1'. Let's discuss this as I  would rather keep the pattern rather than have one exception which is the inverse.

Reply:
I misunderstood what was being asked. This is fine and fixed in 9b6d86e.